### PR TITLE
ReaHTML/CSS & JavaScript capstone project - Conference pagedme file

### PR DESCRIPTION
**Pull request by Feliverse**
### HTML/CSS & JavaScript capstone project - Conference page
**Learning objectives**

1. Use semantic HTML tags.
2. Apply best practices in HTML code.
3. Use CSS selectors correctly.
4. Use CSS box model.
5. Use Flexbox to place elements in the page.
6. Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
7. Use GitHub Pages to deploy web pages.
8. Apply JavaScript best practices and language style guides in code.
9. Use JavaScript to manipulate DOM elements.
10. Use JavaScript events.
11. Use objects to store and access data.
12. Communicate technical concepts to other technical people.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ HTML/CSS & Javascript
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://feliverse.github.io/conference-page/)
 - [Loom Video Presentation](https://www.loom.com/share/ae12d221fbdf498181028d47d3b9fc13)
```
Comment (joskalenda) [inline README.md:115]: - [x] Next time please do not merge a branch before it is reviewed, the reviewer needs to have all your changed files in order to well perform the review, apart from that, your work is to salute, nice job @Feliverse, I just have a few issues to raise on the design before the work gets approved



- This hero text is not using the same design color and font-weight as per the original design, please try adjusting a bit more your image that serves as text color to match the original design on the left
- Kindly use the correct CSS properties as per the requirement, font, padding, color, and all other staff to get the same design as per the original one



- [x] As I pointed out with arrows, please consider making the necessary changes to match the original design, 
- Let's start with the speaker's name, if I believe my eyes, color, and font weight are not the correct ones using the original template, please fix that too
- The paragraph in red should be styled in `italic` font, please update that and adjust the padding between it and the divider half line. 
- The last text is a bit more black, and the font-weight is confusing a bit, please stick to the design



- [x] Please use the same color, font, and padding as per the original design, as shown in the section above, please change the `cc campaign` from black color to red color.



- Please fix this up, the about page text is not the correct one as per the original template
- The paragraph here needs to get a few touches and make look like the one on the original template
- Please make the nav menu wrap the entire page, there is a white space at the bottom of the.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Feliverse , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
| |

## Highlights

- [ ]... ✅ Green linter

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Feliverse) [reply README.md:115]: @joskalenda, the recommendations did it, and the page looks much good.
Thank you

**Review by jerryowusu — APPROVED** [review]
Hi @Feliverse  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

kpop-approved

## Highlights
All linters are passing :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**